versions/draft-3/SPEC.md

```diff
@@ -248,6 +248,9 @@ Float f = 27.3             # A floating point number
 Boolean b = true           # A boolean true/false
 String s = "hello, world"  # A string value
 File f = "path/to/file"    # A file
+FileURI u1 = "gs://bucket/file" # URI for Google cloud storage file
+FileURI u2 = "path/to/file"    # URI for local file
+FileURI u3 = "file:///path/to/file"    # URI for local file 
```

Comment:
I'd only have a single example here and include the other possibilities in the description section below

Reply:
Done